scripts/dumps/prepare_items_dumps

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+source ./scripts/dumps/lib/utils.sh
+
+items_dump="${today_folder}/items.ttl"
+dbHost=$(node -p "require('config').db.fullHost()")
+
+echo "
+@prefix wdt: <http://www.wikidata.org/prop/direct/> .
+@prefix inv: <http://inventaire.io/entity/> .
+@prefix invexamplar: <http://inventaire.io/items/> .
+" > $items_dump
+
+# We only need the items entities for now
+curl -s "${dbHost}/items-prod/_design/items/_view/byEntity" |
+  jq '.rows[] | [.id, .key[0]] | join(" wdt:P1574 ")' |
```

Comment:
arent you pushing item ID in the ttl with this line ?

Reply:
yes, that's why it's [commented-out in `prepare_dumps`](https://github.com/inventaire/inventaire/blob/9a9b8744795c738949fee99d05689576ae4b3a0b/scripts/dumps/prepare_dumps#L4-L14), but maybe that should just be removed(?) Keeping track of it somewhere could be of value though